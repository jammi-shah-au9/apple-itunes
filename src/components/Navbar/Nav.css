#logo {
    color: #d3d3d3;
    font-size: 18px;
    margin-top: 0.5em;
  }
  #logo:hover{
      color:white
  }
  
   nav{
      display: flex;
      justify-content: center;
      line-height: 2.8em;
  }
  
  nav div{
      cursor: pointer;
      text-decoration: none;
      color: #d3d3d3;
      margin-right: 1em; 
      margin-left: 4em;
      font-size: 13px;
  }
  nav div:hover{
      cursor: pointer;
      color: white;
      transition: .3s;
  } 