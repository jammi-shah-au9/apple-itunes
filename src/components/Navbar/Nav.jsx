import React, { Fragment } from "react";
import "antd/dist/antd.css";
import styled from "styled-components";
import "./Nav.css";



const Container159 = styled.div`
  margin-left: 2em;
  margin-right: 6em;
`;

const Nav1 = styled.div`
  background-color: black;
  border: 1px black solid;
  height: 3em;
  position:sticky;
  top: 0;
  z-index:1;
`;

export default function Nav() {
  return (
    <Fragment>
      <Nav1 data-testid='nav'>
        <Container159>
          <nav>
            <div><i id="logo" className="fa fa-apple" aria-hidden="true"></i></div>
            <div>Mac</div>
            <div>iPad</div>
            <div>iPhone</div>
            <div>Watch</div>
            <div>TV</div>
            <div>Music</div>
            <div>Support</div>
            <div>Where to Buy</div>
            <div><i className="fa fa-search" aria-hidden="true"></i></div>
          </nav>
        </Container159>
      </Nav1>
    </Fragment>
  );
}
