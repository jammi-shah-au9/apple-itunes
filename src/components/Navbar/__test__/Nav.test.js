import React from 'react'
import ReactDOM from 'react-dom'
import Nav from '../Nav'
import renderer from 'react-test-renderer'

it("renders without crashing",()=>{
    const div = document.createElement("div")
    ReactDOM.render(<Nav></Nav>,div)
})


it("matches navbar snapshot", ()=>{
    const navtree = renderer.create(<Nav/>).toJSON();
    expect(navtree).toMatchSnapshot();
})