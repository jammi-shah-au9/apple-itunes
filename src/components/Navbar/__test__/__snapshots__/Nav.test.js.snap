// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches navbar snapshot 1`] = `
<div
  className="sc-gtssRu gfLRqM"
  data-testid="nav"
>
  <div
    className="sc-bdnylx bQSunl"
  >
    <nav>
      <div>
        <i
          aria-hidden="true"
          className="fa fa-apple"
          id="logo"
        />
      </div>
      <div>
        Mac
      </div>
      <div>
        iPad
      </div>
      <div>
        iPhone
      </div>
      <div>
        Watch
      </div>
      <div>
        TV
      </div>
      <div>
        Music
      </div>
      <div>
        Support
      </div>
      <div>
        Where to Buy
      </div>
      <div>
        <i
          aria-hidden="true"
          className="fa fa-search"
        />
      </div>
    </nav>
  </div>
</div>
`;
